test/integration/options.spec.js

```diff
@@ -433,7 +433,7 @@ describe('options', function() {
           if (error) {
             return done(error);
           }
-          expect(result.output, 'to contain', 'Usage:');
+          expect(result.output, 'to contain', 'Run tests with Mocha');
```

Comment:
Would prefer to revert this one. The "Usage:" part _should be there_. Fixed above.

Reply:
- [ ] do this

Reply:
This doesn't make sense in yargs:

```plain
mocha [options..] [spec..]

Usage:

Commands
  mocha [options..] [spec..]     Usage:                                [default]
  mocha init <path>              create a client-side Mocha setup at <path>
```

Ideally we'd have a bit more control over how and where the default command displays.  We can ask yargs to do something about it (or propose something) but IMO it shouldn't hold this up.

Reply:
@bcoe, @nexdrew, how do we do this?